app/i18n/locales/ko_KR.json

```diff
@@ -16,6 +16,8 @@
   "loading.screen.connectingToNetworkMessage": "!!!Connecting to network",
   "loading.screen.reconnectingToNetworkMessage": "!!!Network connection lost - reconnecting",
   "loading.screen.syncingBlocksMessage": "!!!Syncing blocks",
+  "profile.languageSelect.form.languageSelectLabel": "!!!Select your language",
+  "profile.languageSelect.form.submitLabel": "!!!Continue",
```

Comment:
There is no need for adding translations manually except when entering real translations. Keys with default translations are managed by `npm run manage-translations`